tools/scripts/upgrades/cosmovisor-start-node.sh

```diff
@@ -1,21 +1,33 @@
 #!/bin/bash
-# This script is intended to run the full node using cosmovisor, and perform an upgrade after the upgrade plan is 
-# submitted on chain. It is helpful to simulate a real network upgrade. If the upgrade includes a consensus-breaking
-# change, make sure the "old" binary does not have these changes (so the real upgrade can be simulated).
-# Make sure to adjust `POKTROLLD_OLD_BINARY_PATH` and `POKTROLLD_NEW_BINARY_PATH`.
 
-# Binaries can be built using `ignite chain build` command and moved to a directory not included in git.
+description="This script is intended to run the full node using cosmovisor, and perform an upgrade after the upgrade plan is \
+submitted on chain. It is helpful to simulate a real network upgrade. If the upgrade includes a consensus-breaking \
+change, make sure the 'old' binary does not have these changes (so the real upgrade can be simulated)."
 
-# Re-requisites:
-# - "Old" binary.
-# - "New" binary.
-# - Cosmovisor. Install with `go install cosmossdk.io/tools/cosmovisor/cmd/cosmovisor@v1.5.0`
-#   Documetation: https://docs.cosmos.network/main/build/tooling/cosmovisor
+echo "$description"
 
+echo -e "\nPre-requisites:"
```

Comment:
NIT: Is it not easier to just do 1 echo statement?

https://stackoverflow.com/a/10969990/768439